README.md

```diff
@@ -1,17 +1,30 @@
 # Morpho Token
 
-This repository contains the Morpho protocol’s ERC20 token.
+This repository contains the Morpho protocol's ERC20 token.
 It is designed to be upgradable and support onchain delegation.
 Additionally, it ships a wrapper contract to simplify the migration of assets from the previous token contract to the new Morpho token contract.
 
-## Upgradability
+## Features
 
-The Morpho token complies with the [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+### Upgradeability
 
-## Delegation
+The Morpho token complies with [EIP-1967](https://eips.ethereum.org/EIPS/eip-1967) to support upgradeability.
+
+### Delegation
 
 The Morpho token supports onchain voting and voting power delegation.
 
+### Role-based permission
```

Comment:
Title and subtitles case is not correct.
```suggestion
### Role-based Permission
```

Reply:
I understand this is standard in english, but I'm still against, this is harder to read IMO

Reply:
> It would be technically not suitable to not capitalize words in a title or subtitle

It's not suitable because of a standard :D
The relevant metric is how well this standard is respected, and how important it is that it is respected. And when it is very much accepted by everyone, and it is important for undertanding (like your example with full-stops), then we say that it's "not technically suitable" or "incorrect" to do something else

We can do whatever we want, but no strong feeling about it

Reply:
There's an interesting reply in the slack. I'll you guys decide.

Reply:
Interesting reply I agree. Let's keep it like that, also because this is what we use everywhere

Reply:
Is it though ? The README here https://github.com/morpho-org/morpho-blue uses title case...

Reply:
You are right, it's not always used. Just sometimes...